features/Context/WebUser.php

```diff
@@ -298,6 +298,7 @@ public function theLocaleShouldBeSelected($locale)
     public function iSwitchTheLocaleTo($locale)
     {
         $this->getCurrentPage()->getElement('Main context selector')->switchLocale($locale);
```

Comment:
why don't we spin instead of waiting?

we should ask the help of Juju if it's too complicated for us